jwst/extract_1d/soss_extract/soss_extract.py

```diff
@@ -1086,23 +1155,25 @@ def run_extract1d(input_model, pastasoss_ref_name,
         if soss_kwargs['subtract_background']:
             log.info('Applying background subtraction.')
             bkg_mask = make_background_mask(scidata, width=40)
-            scidata_bkg, col_bkg, npix_bkg = soss_background(scidata, scimask, bkg_mask=bkg_mask)
+            scidata_bkg, col_bkg = soss_background(scidata, scimask, bkg_mask)
         else:
             log.info('Skip background subtraction.')
             scidata_bkg = scidata
             col_bkg = np.zeros(scidata.shape[1])
 
         # Pre-compute the weights for box extraction (used in modeling and extraction)
         args = (ref_files, scidata_bkg.shape)
-        box_weights, wavelengths = compute_box_weights(*args, width=soss_kwargs['width'])
+        box_weights, wavelengths = _compute_box_weights(*args, width=soss_kwargs['width'])
+        # FIXME: hardcoding the substrip96 weights to unity is a band-aid solution
+        # and needs to be fixed with this PR
```

Comment:
This is still to-do: need to track this down and add a regression test that uses substrip96 data